src/content/docs/apm/transactions/transaction-traces/transaction-traces-trace-details-page.mdx

```diff
@@ -249,23 +367,32 @@ The APM agents have differing rules for when to truncate segments.
 
-Transaction traces that have a lot of segments can be hard to interpret. New Relic helps you by automatically grouping fast and/or repeated method calls into a single segment grouping.
+Things to keep in mind:
 
-For example, you might see a segment titled `7 fast method calls`. This means that, because of their fast completion, New Relic assumed these calls were unlikely to be a problem and grouped them together. You can expand that grouping to see the individual segments.
+* All attributes are grouped into a single `Attributes` section rather than split into the three default attribute types.
+* Segment metric names may differ slightly between distributed traces and transaction traces.
+* Stack traces are not displayed in segment attributes.
+* The trace details page will display slightly different information when you look at a distributed trace instead of a transaction trace. 
 
-Rules underlying segment grouping include:
+## Troubleshoot with segment details [#segment-details]
 
-* **Fast calls:** When New Relic detects four or more consecutive segments with a duration of 7.5 ms or less, these are grouped into a single collapsed header.
-* **Identical calls:** When New Relic detects four or more consecutive identical calls, this usually indicates an N+1 problem. These segments are collapsed. Expanding the collapsed node displays only the first three calls, not all calls.
+When you select a segment, a pane opens up with segment details. These details can help troubleshoot performance issues. 
 
-## Distributed traces [#distributed-traces]
+When you select a segment, segment details will appear. With segment details, you can view: 
 
-When a transaction trace is based on distributed trace data, you can view the full distributed trace by clicking on the distributed trace link in the page header.
+1. Performance tab: 
+  * **Visualize performance trends: charts showcase a segment's operation's average duration and throughput, revealing patterns and potential outliers.
+  * **Compare and contrast**: See how the performance of this specific segment stacks up against the average, pinpointing areas that might need attention.
+
+2. Attributes tab:  
+  * **Explore further**: Understand attribute meanings and potential troubleshooting implications. You can learn more using our [data dictionary](/attribute-dictionary/?dataSource=Distributed+Tracing&event=Span).
 
-The trace details page will display slightly different information when looking at a distributed trace instead of a transaction trace. 
+3. Details tab: 
+  * **Foundational information**: View critical details like the segment's full name, data source, and relevant metadata. You can learn more in our [distributed tracing docs](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui/#prettified-span-names).
+  * **Tailored insights**: Discover segment-specific information crucial for troubleshooting, such as the database query for datastore segments.
```

Comment:
I'd add a tab space in the bullets because if you see the build, they are at the same level that the numbers.